crates/bevy-inspector-egui/src/reflect_inspector/mod.rs

```diff
@@ -1256,7 +1252,7 @@ impl<'a, 'c> InspectorUi<'a, 'c> {
             }
             VariantInfo::Unit(_) => DynamicVariant::Unit,
         };
-        let dynamic_enum = DynamicEnum::new(enum_type_name, variant.name(), dynamic_variant);
```

Comment:
that makes the parameter enum_type_name from the function useless, should we remove it ?